src/apps/umami/arbitrum/umami.compound.token-fetcher.ts

```diff
@@ -52,6 +66,12 @@ export class ArbitrumUmamiCompoundTokenFetcher implements PositionFetcher<AppTok
 
     if (!underlyingToken) return [];
 
+    const data = await axios.get<UmamiApiDatas>('https://horseysauce.xyz/').then(v => v.data);
+
+    const { marinate, mUmamiCompounder } = data;
+    const { apy } = marinate;
+    const { tvl } = mUmamiCompounder;
```

Comment:
You can move this into a separate function and annotate it with `@CacheOnInterval` to call this API less.

See `src/apps/badger/ethereum/badger.vault.token-fetcher.ts` for an example. Apologies we don't have a recipe yet to cover this topic, but effectively:

1. The method is replace by a getting to a cache key
2. The original method is called on the period of time, and cached to that cache key

In effect, even though the token fetcher is run every 45s, the API data can be cached for much longer, and is more resilient to API outages (i.e.: if horseysauce.xyz is down, we can still display stale APY data)

Reply:
Updated, let me know if that's good